examples/golang/actors/http-echo-tinygo/main.go

```diff
@@ -1,26 +1,38 @@
 package main
 
 import (
-	echo "echo/gen"
-	echo_types "echo/gen"
 	"encoding/json"
 	"fmt"
+	echo "github.com/wasmcloud/wasmcloud/examples/golang/actors/echo/gen"
```

Comment:
nit: `golang/actors/http-echo-tinygo/gen` right?

Reply:
Ah yes! I need to change the module name too -- thanks.

Reply:
Fixed!